xml/System/Char.xml

```diff
@@ -110,7 +110,7 @@ The following sections examine the relationship between a <xref:System.Char> obj
 
  Multiple 16-bit code units are used to represent single Unicode characters in the following cases:
 
--   Glyphs, which may consist of a single character or of a base character followed by one or more combining characters. For example, the character ä is represented by a <xref:System.Char> object whose code unit is U+0061 followed by a <xref:System.Char> object whose code unit is U+0308. (The character ä can also be defined by a single <xref:System.Char> object that has a code unit of U+00E4.) The following example illustrates that the character ä consists of two <xref:System.Char> objects.
+-   Glyphs, which may consist of a single character or of a base character followed by one or more combining characters. For example, the character � is represented by a <xref:System.Char> object whose code unit is U+0061 followed by a <xref:System.Char> object whose code unit is U+0308. (The character � can also be defined by a single <xref:System.Char> object that has a code unit of U+00E4.) The following example illustrates that the character � consists of two <xref:System.Char> objects.
```

Comment:
I'm not sure what changed here. There were some encoding issues below, and I was expecting to find something similar here (specifically in the `ä` chars) but I think they remained untouched.